### DISCUSSION TITLE
Help wanted

### DISCUSSION THREAD

**Discussion by MediaBoothKrish** [shlinkio/shlink#2153] [Help wanted]
### Shlink version

4.1.1

### PHP version

8.2.20

### How do you serve Shlink

Self-hosted RoadRunner

### Database engine

MySQL

### Database version

8.2.20

### Summary

Unable to connect app.shlink.io
Oops! Could not connect to this Shlink server.

Make sure you have internet connection, and the server is properly configured and on-line.
<img width="1715" alt="Screenshot 2024-07-08 at 11 10 17 PM" src="https://github.com/shlinkio/shlink/assets/136290746/e87d01c0-99b6-4658-8ce2-0e977620889d">

**Comment by steffen0377** [comment]
Hello!

I think, i have the same problem, and i think, in my installation i'm the problem :-)

Lets say, my shlink.io-Server listens to "sh.mydomain.com", DNS is configured, i can reach the server from internal and external with http://sh.mydomain.com:8080. Yes, i know, unencrypted is not the best way, but one step after the other :-)
This works, i get a 404 Error Page (from internal and external). So i think, roadrunner is working, but isn't right configured.
![image](https://github.com/user-attachments/assets/2d6ab2f7-619f-4b15-8cec-538414da5a0c)

And here start my problems, because i think, there isn't a good documentation or explanation, how to configure.
Where are the Roadrunner ErrorLogs?
What has i to set for 'default_domain'? sh.mydomain.com or www.mydomain.com?
What has i to fill at https://app.shlink.io?
Name: Anything i want ('MyServer1')
URL: http://sh.mydomain.com:8080
API-Key: MyAPIKey

Sorry for this beginners-questions....

Greets
Steffen
